.github/actions/split-tests/package.json

```diff
@@ -11,6 +11,10 @@
   "keywords": [],
   "author": "",
   "license": "MIT",
+  "engines": {
+    "node": ">=16",
+    "pnpm": ">=8"
```

Comment:
@RensR you said the test splitter was failing on 9 right?

Reply:
This one is the only one that doesn't work with the newer engines. I'm actively trying to remove Hardhat, but that seems hard since Automation is still using it. When all non-Automation teams remove their last hardhat I think we can remove the split tests and just have a single hardhat runner. That solves this annoying old node/pnpm version issue